src/Elm/Parser/Tokens.elm

```diff
@@ -197,17 +196,17 @@ multiLineStringLiteral : Core.Parser String
 multiLineStringLiteral =
     Core.succeed identity
         |. Core.symbol "\"\"\""
-        |= Core.loop [] multiLineStringLiteralStep
+        |= Core.loop "" multiLineStringLiteralStep
 
 
-multiLineStringLiteralStep : List String -> Core.Parser (Step (List String) String)
-multiLineStringLiteralStep partsSoFarReverse =
+multiLineStringLiteralStep : String -> Core.Parser (Step String String)
+multiLineStringLiteralStep stringSoFar =
     Core.oneOf
         [ Core.symbol "\"\"\""
-            |> Core.map (\() -> Done (List.Extra.listReverseThenStringConcatFast partsSoFarReverse))
+            |> Core.map (\() -> Done stringSoFar)
         , Core.symbol "\""
-            |> Core.mapChompedString (\v () -> Loop (v :: partsSoFarReverse))
-        , Core.succeed (\v -> Loop (String.fromChar v :: partsSoFarReverse))
+            |> Core.mapChompedString (\v () -> Loop (stringSoFar ++ v))
```

Comment:
`Core.symbol xyz |> Core.mapChompedString (\v () -> ...)` can also be replaced by `Core.string xyz |> Core.map (\v -> ...)` if I'm not mistaken. (Hey, `Simplify` idea :sweat_smile: )

Reply:
What's Code.string and why am I not finding it...

Reply:
Oh no, my bad, it was `Combine.string`, and I removed it in a previous PR. Carry on, nothing to see...